diff --git a/node_modules/@acala-network/types/index.d.ts b/node_modules/@acala-network/types/index.d.ts
index 330dde5..aab5b67 100644
--- a/node_modules/@acala-network/types/index.d.ts
+++ b/node_modules/@acala-network/types/index.d.ts
@@ -1,5 +1,4 @@
 import '@polkadot/types/lookup';
-import './augment/api';
 import './lookup/acala';
 import type { DefinitionRpc, DefinitionRpcSub, OverrideBundleType, OverrideModuleType, RegistryTypes } from '@polkadot/types/types';
 export * as lookupTypes from './lookup/acala';
diff --git a/node_modules/@acala-network/types/index.js b/node_modules/@acala-network/types/index.js
index 6884ffd..54a1a5c 100644
--- a/node_modules/@acala-network/types/index.js
+++ b/node_modules/@acala-network/types/index.js
@@ -25,7 +25,6 @@ var __importStar = (this && this.__importStar) || function (mod) {
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.signedExtensions = exports.typesBundle = exports.typesAlias = exports.rpc = exports.types = exports.lookupTypes = void 0;
 require("@polkadot/types/lookup");
-require("./augment/api");
 require("./lookup/acala");
 const type_definitions_1 = require("@acala-network/type-definitions");
 exports.lookupTypes = __importStar(require("./lookup/acala"));
