diff --git a/node_modules/@galacticcouncil/sdk/dist/types/client/polkadotApi.d.ts b/node_modules/@galacticcouncil/sdk/dist/types/client/polkadotApi.d.ts
index ad31995..e22679d 100644
--- a/node_modules/@galacticcouncil/sdk/dist/types/client/polkadotApi.d.ts
+++ b/node_modules/@galacticcouncil/sdk/dist/types/client/polkadotApi.d.ts
@@ -5,7 +5,6 @@ import type { AssetMetadata } from '@polkadot/types/interfaces';
 import type { Balance } from '@polkadot/types/interfaces/runtime';
 import type { Struct } from '@polkadot/types-codec';
 import type { PoolToken } from '../types';
-import '@polkadot/api-augment';
 interface AssetDetail extends Struct {
     readonly name: string;
     readonly assetType: string;
